GVFS/GVFS.Common/Prefetch/Git/PathWithMode.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace GVFS.Common.Prefetch.Git
+{
+    public class PathWithMode
+    {
+        public PathWithMode(ushort mode, string path)
+        {
+            this.Mode = mode;
+            this.Path = path;
+        }
+
+        public ushort Mode { get; set; }

```

Comment:
Remove these setters, this class should be immutable (and I don't see anything other the constructor setting these properties currently anyway)